### PR TITLE
Add description and examples to Service Bus overview

### PR DESCRIPTION
**Pull request by anthonychu**
@cecilphillip @JeremyLikness

### CODE REVIEW

**Comment by openpublishbuild** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/index.yml](https://github.com/anthonychu/azure-docs-sdk-dotnet/blob/update-servicebus-overview/api/overview/azure/index.yml) | :x:Error | | [Details](#user-content-api%2Foverview%2Fazure%2Findex.yml)
[]() | :x:Error | | [Details](#user-content-)

<a name="api%2Foverview%2Fazure%2Findex.yml"></a>
### [api/overview/azure/index.yml](https://github.com/anthonychu/azure-docs-sdk-dotnet/blob/update-servicebus-overview/api/overview/azure/index.yml)
 - **[Error]** Error transforming model &quot;W:\ochs-o-unknown\raw-\32h4cfm4.bkz\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

<a name=""></a>
### []()
 - **[Error]** Error transforming model &quot;W:\ochs-o-unknown\raw-\32h4cfm4.bkz\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C2%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201708012355180519-282%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api/overview/azure/service-bus.md [hunk 1]
```diff
@@ -25,10 +29,24 @@ Install the [NuGet package](https://www.nuget.org/packages/WindowsAzure.ServiceB
...
 ```
 
+### Code Example
+
+This example sends a message to a Service Bus queue.
+
+```csharp
+// using Microsoft.ServiceBus.Messaging;
+
+QueueClient client = QueueClient.CreateFromConnectionString(connectionString, queueName);
```
Comment (cecilphillip) [inline api/overview/azure/service-bus.md]: Should this be wrapped in a using statement? Asking based on the style of the other code samples

File: api/overview/azure/service-bus.md [hunk 2]
```diff
@@ -43,10 +61,31 @@ Install the [NuGet package](https://www.nuget.org/packages/Microsoft.Azure.Manag
...
+    QueueInner parameters = new QueueInner
+    {
+        MaxSizeInMegabytes = 1024
+    };
+    await client.Queues.CreateOrUpdateAsync(resourceGroupName, namespaceName, queueName, parameters);
+}
+```
+
 > [!div class="nextstepaction"]
 > [Explore the management APIs](/dotnet/api/overview/azure/servicebus/management)
```
Comment (cecilphillip) [inline api/overview/azure/service-bus.md]: I think you need to add these to the end of the overview

[PackageManager]: https://docs.microsoft.com/nuget/tools/package-manager-console  
[DotNetCLI]: https://docs.microsoft.com/dotnet/core/tools/dotnet-add-package

File: api/overview/azure/service-bus.md (hunk 1 above)
Comment (anthonychu) [inline api/overview/azure/service-bus.md]: Good point. I'll see if it implements `IDisposable`.

File: api/overview/azure/service-bus.md (hunk 2 above)
Comment (anthonychu) [inline api/overview/azure/service-bus.md]: They appear to be there already.

**Comment by anthonychu** [comment]
@JeremyLikness 

The only samples I found were ones like these. I'm not sure if they're useful enough to be in a bulleted list: https://azure.microsoft.com/en-us/resources/samples/service-bus-dotnet-manage-publish-subscribe-with-advanced-features/

It's also a bit tricky linking to all of them as they don't seem to be properly tagged as .NET. Do you have thoughts on how to fix that? https://azure.microsoft.com/en-us/resources/samples/?term=service+bus

Good point on Acrolinx as I forgot to run it. It's 93. I'll push a change in a sec that'll push it to 100. 😄

File: api/overview/azure/service-bus.md (hunk 1 above)
Comment (anthonychu) [inline api/overview/azure/service-bus.md]: It doesn't implement `IDisposable`.  I guess it's a factory method so they can manage the lifetime on our behalf.

**Comment by openpublishbuild** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/index.yml](https://github.com/anthonychu/azure-docs-sdk-dotnet/blob/update-servicebus-overview/api/overview/azure/index.yml) | :x:Error | | [Details](#user-content-api%2Foverview%2Fazure%2Findex.yml)
[]() | :x:Error | | [Details](#user-content-)

<a name="api%2Foverview%2Fazure%2Findex.yml"></a>
### [api/overview/azure/index.yml](https://github.com/anthonychu/azure-docs-sdk-dotnet/blob/update-servicebus-overview/api/overview/azure/index.yml)
 - **[Error]** Error transforming model &quot;W:\j41f-o-2b001d62\raw-\4d4ztmig.mye\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

<a name=""></a>
### []()
 - **[Error]** Error transforming model &quot;W:\j41f-o-2b001d62\raw-\4d4ztmig.mye\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C2%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201708021643055698-282%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by JeremyLikness** [comment]
@anthonychu for the samples I'd just pick out the top 5 and make a bulleted list, starting with the first one you mentioned and followed by four you pick from the pages - just direct links into them and the links have their title. Make sense?

**Comment by anthonychu** [comment]
@JeremyLikness Removed .NET Core CLI from client. Added samples.

**Comment by openpublishbuild** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/index.yml](https://github.com/anthonychu/azure-docs-sdk-dotnet/blob/update-servicebus-overview/api/overview/azure/index.yml) | :x:Error | | [Details](#user-content-api%2Foverview%2Fazure%2Findex.yml)
[]() | :x:Error | | [Details](#user-content-)

<a name="api%2Foverview%2Fazure%2Findex.yml"></a>
### [api/overview/azure/index.yml](https://github.com/anthonychu/azure-docs-sdk-dotnet/blob/update-servicebus-overview/api/overview/azure/index.yml)
 - **[Error]** Error transforming model &quot;W:\op0c-o-e3661e7b\raw-\zekzksvt.ukk\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

<a name=""></a>
### []()
 - **[Error]** Error transforming model &quot;W:\op0c-o-e3661e7b\raw-\zekzksvt.ukk\rawmodel\api\overview\azure\index.raw.json&quot; generated from &quot;api/overview/azure/index.yml&quot; using &quot;ManagedReference.html.primary.js&quot;. Error running Transform function inside template preprocessor: 

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C2%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201708022028126377-282%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by openpublishbuild** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api/overview/azure/service-bus.md](https://github.com/anthonychu/azure-docs-sdk-dotnet/blob/update-servicebus-overview/api/overview/azure/service-bus.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/dotnet/api/overview/azure/service-bus?branch=pr-en-us-282) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C4%5C14458122-e42f-10e3-bd3d-ee1c7494cfbf%5CPullRequest%5C201708040326233395-282%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by JeremyLikness** [comment]
#sign-off

**Comment by CamSoper** [comment]
@anthonychu Is there not any way to do Service Bus from .NET Core?

**Comment by cecilphillip** [comment]
There is [this NuGet package](https://www.nuget.org/packages/Microsoft.Azure.ServiceBus) but it's in preview. Just took a look at the [management package](https://www.nuget.org/packages/Microsoft.Azure.Management.ServiceBus/) and that one supports netstandard 1.4.

**Comment by CamSoper** [comment]
@cecilphillip Ahhhh, gotcha.  Thanks!

**Comment by anthonychu** [comment]
The new client is now in RC and should GA soon. We'll need to update this when that happens.

**Pull request merged**